src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -64,6 +64,14 @@
      */
     private Set<Block> freeBlocks = new HashSet<>();
 
+    /**
+     * The Set of waypoints.
+     */
+    private Set<BlockPos> wayPoints = new HashSet<>();
+
+    /**
+     * The overall happiness of the colony.
+     */
     private double overallHappiness = 5;
```

Comment:
Best would be to make a util.constrants class with these game changing constants and use that instead of all the 5 literals. Would be good to start with that